### PR TITLE
Feature/soss dds requirements

### PR DESCRIPTION
**Pull request by lemunozm**
CMake needed changes to compile the core-standalone version

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/287/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/262/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/285/)

File: include/dds/core/xtypes/detail/PrimitiveTypes.hpp [hunk 1]
```diff
@@ -66,6 +66,8 @@ struct dynamic_type_traits<uint32_t>
     static constexpr const char* NAME = "uint32_t";
 };
 
+struct AAAAA : public dynamic_type_traits<uint32_t> {};
```
Comment (laura-eprosima) [inline include/dds/core/xtypes/detail/PrimitiveTypes.hpp]: Are you sure you want to leave the name like that?

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/305/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/280/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/303/)

File: include/dds/core/xtypes/detail/PrimitiveTypes.hpp (hunk 1 above)
Comment (lemunozm) [reply include/dds/core/xtypes/detail/PrimitiveTypes.hpp]: Upss, this line should be deleted

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/316/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/291/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/314/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/324/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/297/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/322/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/329/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/302/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/327/)

**Pull request merged**